mm2src/mm2_main/src/mm2.rs

```diff
@@ -303,7 +303,7 @@ pub fn mm2_main(version: String, datetime: String) {
 /// Parses and returns the `first_arg` as JSON.
 /// Attempts to load the config from `MM2.json` file if `first_arg` is None
 pub fn get_mm2config(first_arg: Option<&str>) -> Result<Json, String> {
-    let conf_path = common::kdf_config_file();
+    let conf_path = common::kdf_config_file().map_err(|e| e.to_string())?;
     let conf_from_file = slurp(&conf_path);
     let conf = match first_arg {
         Some(s) => s,
```

Comment:
Wouldn't returning an early error lead to config from command line not working?

Reply:
Didn't even know that was possible.